packages/node_modules/@webex/internal-plugin-ediscovery/test/unit/spec/content.js

```diff
@@ -21,8 +21,6 @@ describe('EDiscovery Content API Tests', () => {
 
   describe('GetContent Tests', () => {
     it('GetContent succeeds', async () => {
-      // getContent calls getContentSummary internally, so we need to stub it here
-      webex.internal.ediscovery.getContentSummary = sinon.stub().returns(Promise.resolve());
```

Comment:
No longer necessary thankfully

Reply:
For for own curiosity, why is it no longer needed? Did it make a network request before and now it doesn't?

Reply:
GetContent had a baked in call to getContentSummary in order to populate the cache, this was over complicating the flow so we've removed it from the plugin and now control it from the client